go/libraries/doltcore/sqle/sqlinsert_test.go

```diff
@@ -57,30 +58,54 @@ var systemTableInsertTests = []InsertTest{
 		Name: "insert into dolt_query_catalog",
 		AdditionalSetup: CreateTableFn(doltdb.DoltQueryCatalogTableName,
 			DoltQueryCatalogSchema,
-			NewRow(types.String("existingEntry"), types.Uint(2), types.String("example"), types.String("select 2+2 from dual"), types.String("description"))),
+			NewRowWithSchema(row.TaggedValues{
+				doltdb.QueryCatalogIdTag:          types.String("existingEntry"),
+				doltdb.QueryCatalogOrderTag:       types.Uint(2),
+				doltdb.QueryCatalogNameTag:        types.String("example"),
+				doltdb.QueryCatalogQueryTag:       types.String("select 2+2 from dual"),
+				doltdb.QueryCatalogDescriptionTag: types.String("description")},
+				DoltQueryCatalogSchema)),
 		InsertQuery: "insert into dolt_query_catalog (id, display_order, name, query, description) values ('abc123', 1, 'example', 'select 1+1 from dual', 'description')",
 		SelectQuery: "select * from dolt_query_catalog",
 		ExpectedRows: CompressRows(DoltQueryCatalogSchema,
-			NewRow(types.String("abc123"), types.Uint(1), types.String("example"), types.String("select 1+1 from dual"), types.String("description")),
-			NewRow(types.String("existingEntry"), types.Uint(2), types.String("example"), types.String("select 2+2 from dual"), types.String("description")),
+			NewRowWithSchema(row.TaggedValues{
```

Comment:
Because these are result set rows, you shouldn't need to provide tags here. Result set schemas should continue to use the counting up from 0 scheme they did before. Am I missing something?

Reply:
reverted to the simplified result form